customization-base/README.md

```diff
@@ -181,14 +181,14 @@ with customization
 public void customize(LibraryCustomization customization, Logger logger) {
     PackageCustomization models = customization.getPackage("com.azure.myservice.models");
     ClassCustomization foo = models.getClass("Foo");
-    foo.addAnnotation("JsonClassDescription(\"Foo class\")");
+    foo.addAnnotation("Deprecated");
 }
 ```
 
 will generate
 
 ```java readme-sample-add-class-annotation-result
-@JsonClassDescription("Foo class")
```

Comment:
Why is `@JsonClassDescription` replaced with `@Deprecated`?

Reply:
This was to get Jackson removed as a dependency, I can revert this by using Jackson as a test dependency. But with our overall shift on limiting the usage of Jackson, I thought to remove using it as an example as well.